src/foundry/client/pixi/layers/placeables/lighting.d.ts

```diff
@@ -0,0 +1,141 @@
+export {};
+
+/**
+ * The Lighting Layer which ambient light sources as part of the CanvasEffectsGroup.
+ */
+declare global {
+  class LightingLayer extends PlaceablesLayer<"AmbientLight"> {
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    static get instance(): Canvas["lighting"];
+
+    static override documentName: "AmbientLight";
+
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    override options: LightingLayer.LayerOptions;
+
+    /**
+     * @defaultValue
+     * ```
+     * foundry.utils.mergeObject(super.layerOptions, {
+     *  name: "lighting",
+     *  rotatableObjects: true,
+     *  zIndex: 300
+     * })
+     * ```
+     */
+    static override get layerOptions(): LightingLayer.LayerOptions;
+
+    override get hookName(): string;
+
+    override _activate(): void;
+
+    protected override _onDragLeftStart(event: PIXI.FederatedEvent): ReturnType<AmbientLight["draw"]>;
+
+    protected override _onDragLeftMove(event: PIXI.FederatedEvent): Promise<void>;
+
+    protected override _onDragLeftCancel(event: PointerEvent): Promise<void>;
+
+    protected _onMouseWheel(event: WheelEvent): void;
+
+    /**
+     * Actions to take when the darkness level of the Scene is changed
+     * @param darkness - The new darkness level
+     * @param prior    - The prior darkness level
+     */
+    protected _onDarknessChange(darkness: number, prior: number): void;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#background has been refactored to EffectsCanvasGroup#background"
+     */
+    get background(): EffectsCanvasGroup["background"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#illumination has been refactored to EffectsCanvasGroup#illumination"
+     */
+    get illumination(): EffectsCanvasGroup["illumination"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#channels has been refactored to EffectsCanvasGroup#lightingChannelColors"
+     * @remarks lightingChannelColors was removed in v11 without this deprecation being updated
+     */
+    get channels(): undefined;
```

Comment:
The version of Foundry I'm checking, which could be wrong, doesn't return undefined. It is getting deprecated but it should probably be fixed if it matters at all.

Reply:
I checked in v11.315 and got `undefined`, because the property it's trying to access as a shim was inadvertently removed.